test/core/derivatives/implementation/backpack.py

```diff
@@ -36,20 +37,16 @@ def jac_t_mat_prod(self, mat):  # noqa: D102
             self.problem.module, None, None, mat
         )
 
-    def weight_jac_t_mat_prod(self, mat, sum_batch, subsampling=None):  # noqa: D102
+    def param_jac_t_mat_prod(
```

Comment:
Rename into `param_mjp` to improve consistency

Reply:
Yes.